scala/SimpleMlp/README.md

```diff
@@ -0,0 +1,27 @@
+# SimpleMlp
+
+This is an example about how to use [Analytics Zoo](https://github.com/intel-analytics/analytics-zoo) to build your own deep learning application. You can use both maven or sbt to create your project and add analytics-zoo to your dependency, see pom.xml and build.sbt for details.   
+
+## Build
+### 1. Build with maven
+```bash
+mvn clean package
+```
+
+### 2. Build with sbt
+```
+sbt assembly
+```
+
+## Run
+### 1. Run with Spark-submit
+```bash
+spark-submit \
+  --master local[4] \
+  --class com.github.qiuxin2012.SimpleMlp \
```

Comment:
change to `com.intel.analytics.zoo.tutorial.SimpleMlp`?

Reply:
fixed in #37